core/src/main/scala/org/apache/pekko/persistence/jdbc/state/scaladsl/JdbcDurableStateStore.scala

```diff
@@ -117,7 +117,7 @@ class JdbcDurableStateStore[A](
     db.run(queries.deleteFromDb(persistenceId).map(_ => Done))
```

Comment:
Can you add a `deprecated` comment here？

Reply:
the super class method is deprecated so this is too - no need to also deprecate this too